packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -29,6 +45,25 @@ const { densityClass } = useDensity(props);
         ]"
       >
         <slot></slot>
+
+        <!-- info: we can't use tbody here because we determine via CSS
+         whether to show "empty" when no tbody is present -->
+        <div class="onyx-table__empty">
```

Comment:
I think a <div> is not valid HTML for a table since a table
![image](https://github.com/SchwarzIT/onyx/assets/67898185/3f5ba3a3-93f4-45ee-8d68-5eeea463d747)